app/models/division.rb

```diff
@@ -50,8 +50,7 @@ def self.root_internal_name
 
   def self.root
     # create on demand if not present for convenience of blank db's and test cases
-    # @@root ||=
```

Comment:
It looks like the closure_tree method will potentially either return an arbitrary root node.  I wanted a well defined root node.  

Regarding find_or_create_by() followed by an update_attribute(), won't that always perform the update operation?  I'd rather avoid the update when it already exists, which should be almost always.